### PR TITLE
John_Masters_Variable_Homework

### PR DESCRIPTION
**Pull request by werewolfwun**

### CODE REVIEW

File: VariablesHomework/VariablesHomework.cs [hunk 1]
```diff
@@ -8,50 +8,56 @@ public class Variables
         [TestMethod]
         public void Declaration()
         {
-            Assert.IsTrue(number == 25); // Make this pass
-            Assert.IsTrue(boolean == true); // Make this pass
-            Assert.IsTrue(decimalValue == 99.9); // Make this pass
-            Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
-            Assert.IsTrue(words == "I am a programmer now"); // Make this pass
-            Assert.IsTrue(character == 'z'); // Make this pass
-        }
```
Comment (csevier) [inline VariablesHomework/VariablesHomework.cs:17]: Hey John, 

This was not the intent of the homework. The Assert.IsTrue statements need to remain as they are the questions.

File: VariablesHomework/VariablesHomework.cs [hunk 2]
```diff
@@ -148,7 +154,7 @@ public void Conversion()
         {
             int i = 13;
             double d = i;
-            Assert.IsTrue(d == /*Answer here*/); // Make this pass
+            Assert.IsTrue(d ==  // Make this pass
```
Comment (csevier) [inline VariablesHomework/VariablesHomework.cs:157]: This will not compile, or run or pass.

**Review by csevier — CHANGES_REQUESTED** [review]
The intent of the homework is to have a compiling  projects whos asserts pass the check.

**Comment by werewolfwun** [comment]
Thanks, Cody.  Working on it now.

Grandkids are gone so I can concentrate.

I talked to John Beach and he gave me some pointers so I am going to try
those.

I have three asserts that are not working and I am trying to debug why.

[image: image.png]

Also, I wanted to let you know the coming week is my last week of work at
my company.  I am resigning from that job (I make more with my Army
Retirement and disability anway 😀)so I can focus full time on this course.

The point is I may not make all the classes the coming week but will
everyone after that, barring misfortune.

Thanks for the review.  I will resubmit later tonight.

**Comment by werewolfwun** [comment]
[image: image.png]

**Review by werewolfwun — COMMENTED** [review]
I resubmitted my homework with the changes

**Comment by werewolfwun** [comment]
Hey Cody, can I ask you a couple of questions?

Best,

**Pull request closed without merging**